### ISSUE TITLE
Error : Installation PlatformIO Core

### ISSUE THREAD

**Issue by edwin6938** [Microsoft/vscode#41403]
- VSCode Version: 1.19.1
- OS Version: Windows 7 Enterprise

Steps to Reproduce:

1. I tried to install PlatformIO Core
2. After few second I am asked if python 2.7 is already install. If so, enter the path... That's what I do
3. After few second, I receive a msgbox error (cf. enclosed)
![capture](https://user-images.githubusercontent.com/7591024/34778319-efb3ded4-f614-11e7-8a0b-aef8525ec9f0.PNG)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by aeschli** [comment]
@edwin6938 Please file the issue against the extension.

**Comment by edwin6938** [comment]
I dont understand?

**Comment by edwin6938** [comment]
@aeschli i dont understand?

**Comment by aeschli** [comment]
VSCode has no feature related PlatformIO Core. You have to file the bug report against a different product, or maybe it's an extension of VSCode?

**Comment by edwin6938** [comment]
Yes this is in extension of vs code. In vscode, on the left I try to
"download" this extension as well you have python, C... extensions

Le 11 janv. 2018 5:52 PM, "Martin Aeschlimann" <notifications@github.com> a
écrit :

**Comment by aeschli** [comment]
Which extension? Is it 'Platform IDE' ?

**Comment by edwin6938** [comment]
This is platformio core. It s an ide. That an officiel version I found in
extension. And it seems to me python is not found because I was warned that
python 2.7 is not found.
I tried to give the path but nothing work

Le 12 janv. 2018 10:04, "Martin Aeschlimann" <notifications@github.com> a
écrit :

Which extension? Is it 'Platform IDE' ?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Microsoft/vscode/issues/41403#issuecomment-357181826>,
or mute the thread
